settings.gradle

```diff
@@ -18,14 +18,14 @@ include ':persist-native'
 include 'persist-compiler-plugin'
 include ':persist-ballerina'
 include 'persist-compiler-plugin-test'
-include ':persist-examples'
+//include ':persist-examples'
 
 project(':checkstyle').projectDir = file("build-config${File.separator}checkstyle")
 project(':persist-native').projectDir = file('native')
 project(':persist-compiler-plugin').projectDir = file('compiler-plugin')
 project(':persist-ballerina').projectDir = file('ballerina')
 project(':persist-compiler-plugin-test').projectDir = file('compiler-plugin-test')
-project(':persist-examples').projectDir = file('examples')
+//project(':persist-examples').projectDir = file('examples')
```

Comment:
Examples build disabled until https://github.com/ballerina-platform/ballerina-standard-library/issues/3469. is resolved